### DISCUSSION TITLE
ingest SRT stream with passphrase

### DISCUSSION THREAD

**Discussion by poreztomas** [ant-media/Ant-Media-Server#6221] [General]
Hey,
is there any way I can ingest an SRT stream to Ant media that contains passphrase? In Antmedia settings I only see the option to create a livestream and define a stream ID for it. But the signal I need to receive also has a password / passphrase.

thanks for help

<img width="731" alt="Screenshot 2024-03-23 at 11 35 43" src="https://github.com/ant-media/Ant-Media-Server/assets/164624443/b5932496-6dc8-4f89-8301-6556f00a1337">

<img width="1165" alt="Screenshot 2024-03-23 at 11 37 15" src="https://github.com/ant-media/Ant-Media-Server/assets/164624443/65ccc512-49f7-4170-a2a0-f2365a3afb8d">

**Comment by burak-58** [comment]
Hi @poreztomas,
Thanks for your question. Let me convert this to github discussion so that everyone can follow it.

**Comment by Mohit-3196** [comment]
Hi @poreztomas,
How are you?

You can use stream security to secure the SRT streams with the help of tokens. On the server side, AMS supports tokens to secure the streams and it works for the SRT as well https://antmedia.io/docs/guides/publish-live-stream/srt/#publishing-srt-stream-with-obs

We have different token mechanisms available to secure the streams and you can check them in detail from the stream security document here https://antmedia.io/docs/category/stream-security/

Thank you.
